src/components/map/Map.tsx

```diff
@@ -36,18 +38,29 @@ type Props = {
 const MARKER_ICON_SIZE = normalize(40);
 const CIRCLE_SIZES = [60, 50, 40, 30];
 
-const MapIcon = ({
-  iconColor,
+export const MapIcon = ({
+  iconColor = colors.lighterPrimary,
   iconName = 'location',
-  iconSize = MARKER_ICON_SIZE
+  iconSize = MARKER_ICON_SIZE,
+  iconStrokeColor = colors.darkerPrimary,
+  iconStrokeWidth = 1.5
 }: {
   iconColor?: string;
   iconName?: string;
   iconSize?: number;
+  iconStrokeColor?: string;
+  iconStrokeWidth?: number;
 }) => {
   const MarkerIcon = Icon[_upperFirst(iconName) as keyof typeof Icon];
 
-  return <MarkerIcon color={iconColor} size={iconSize} />;
+  return (
+    <MarkerIcon
+      color={iconColor}
+      size={iconSize}
+      strokeColor={iconStrokeColor}
+      strokeWidth={iconStrokeWidth}
+    />
+  );
```

Comment:
the icon in the marker is not positioned correctly for Sue. Instead it is added to the anchor section.

![Simulator Screenshot - iPhone 15 Pro Max - 2024-07-29 at 09 44 40](https://github.com/user-attachments/assets/2c8567c1-c8f6-4deb-b0ba-31f604f8b2af)

Reply:
will be adjusted

![Uploading Simulator Screenshot - iPhone 15 Plus - 2024-09-20 at 17.36.55.png…]()